applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/app.overlay

```diff
@@ -52,3 +52,18 @@
 		};
 	};
 };
+
+&cpuapp_rw_partitions {
```

Comment:
Shouldn't we configure it somewhat similar to the SMP sample?

https://github.com/nrfconnect/sdk-nrf/blob/main/samples/suit/smp_transfer/sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi

?

BTW, where is located the storage partition?

Reply:
discussed on Teams